scripts/upgrade

```diff
@@ -28,7 +28,7 @@ db_name=$(ynh_app_setting_get $app db_name)
 # CHECK VERSION
 #=================================================
 
-ynh_abort_if_up_to_date
+upgrade_type=$(ynh_check_app_version_changed)
```

Comment:
Is it useful as it's not used later on...?

Reply:
Not really I know, because the app itself isn't upgrade by the script. But I'd rather prefer to have this variable filled in case we would like to use it later.